demo/src/app/components/+timepicker/demos/arrowkeys/arrowkeys.ts

```diff
@@ -0,0 +1,10 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'demo-timepicker-arrowkeys',
+  templateUrl: './arrowkeys.html'
+})
+export class DemoTimepickerArrowkeysComponent {
+  allowArrowKeys = true;
+  myTime: Date = new Date();
```

Comment:
remove type annotation 
if type annotation isn't provided, typescript will use the type of the first assigned value